vhdbuilder/packer/imagecustomizer/azlosguard/azlosguard.yml

```diff
@@ -0,0 +1,307 @@
+      destination: /etc/systemd/system/measure-tls-bootstrapping-latency.service
+      permissions: 600
+    # aks log collector
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.sh
+      destination: /opt/azure/containers/aks-log-collector.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector-send.py
+      destination: /opt/azure/containers/aks-log-collector-send.py
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.service
+      destination: /etc/systemd/system/aks-log-collector.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.slice
+      destination: /etc/systemd/system/aks-log-collector.slice
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-log-collector.timer
+      destination: /etc/systemd/system/aks-log-collector.timer
+      permissions: 600
+    # aks check network
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-check-network.sh
+      destination: /opt/azure/containers/aks-check-network.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-check-network.service
+      destination: /etc/systemd/system/aks-check-network.service
+      permissions: 600
+    # aks logrotate
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/aks-logrotate-override.conf
+      destination: /etc/systemd/system/logrotate.timer.d/override.conf
+      permissions: 644
+    # nftables
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables
+      destination: /etc/systemd/system/ipv6_nftables
+      permissions: 644
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables.service
+      destination: /etc/systemd/system/ipv6_nftables.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/ipv6_nftables.sh
+      destination: /opt/scripts/ipv6_nftables.sh
+      permissions: 744
+    # kms
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/kms.service
+      destination: /etc/systemd/system/kms.service
+      permissions: 600
+    # containerd
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/containerd.service
+      destination: /etc/systemd/system/containerd.service
+      permissions: 600
+    # localdns
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns.service
+      destination: /etc/systemd/system/localdns.service
+      permissions: 600
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns.sh
+      destination: /opt/azure/containers/localdns.sh
+      permissions: 755
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/localdns-delegate.conf
+      destination: /etc/systemd/system/localdns.service.d/delegate.conf
+      permissions: 644
+
+  services:
+    enable:
```

Comment:
you're missing `secure-tls-bootstrap` here, and its corresponding unit file: https://github.com/Azure/AgentBaker/blob/master/parts/linux/cloud-init/artifacts/secure-tls-bootstrap.service

Reply:
Thanks! Added the service file to the image build but did not set it to enabled by default since [cse_main.sh](https://github.com/Azure/AgentBaker/blob/8e0dda6e7c42939d57fb21a318e1fa6c4c396d90/parts/linux/cloud-init/artifacts/cse_main.sh#L294) will enable and start it if ENABLE_SECURE_TLS_BOOTSTRAPPING=true.